src/features/rewards/walletWrapper/index.tsx

```diff
@@ -50,6 +50,7 @@ export interface Props {
     onAlertClose?: () => void
   } | null
   id?: string
+  contentPadding?: boolean
```

Comment:
Can we provide flag `compact` instead of `contentPadding`. We can use this flag to remove min-height as well, which is a problem currently as well

Reply:
++